CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing Guidelines
+
+We love improvements to our documentation! Our process for accepting changes has a few steps:
+
+1. If you haven't submitted anything before, and you aren't (yet!) a member of our organization, **fork and clone** the repo:
+
+        $ git clone git@github.com:<your-username>/<repository-name>.git
+
+  Organization members should clone the upsteam repo, instead of working from a personal fork:
+
+      $ git clone git@github.com:datarefuge/<repository-name>.git
+
+2. Create a **new branch** for the changes you want to work on. Choose a topic for your branch name that reflects the change:
+
+        $ git checkout -b <branch-name>
+
+3. **Create or modify the files** with your changes. If you want to show other people work that isn't ready to merge in, commit your changes then create a pull request (PR) with _WIP_ or _Work In Progress_ in the title.
+
+        https://github.com/edgi-govdata-archiving/<repository-name>/pull/new/master
+
+4. Once your changes are ready for final review, commit your changes then modify or **create your pull request (PR)**, assign as a reviewer or ping (using "`@<username>`") a Lieutenant (someone able to merge in PRs) active on the project (all Lieutenants can be pinged via `@datarefuge/lieutenants`)
```

Comment:
I listed my name and Laurie's for now. Should I list your name too, @dcwalk, to have at least one person from EDGI?